cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -349,12 +373,13 @@ public static OrderedProperties getOrderedProperties(String configFileName) {
     }
 
     /**
-     * Transforms map to log4j properties file format
-     * @param properties map with properties
-     * @return
+     * Transforms map to log4j properties file format.
+     *
+     * @param properties map of log4j properties.
+     * @return log4j properties as a String.
      */
     protected static String createPropertiesString(OrderedProperties properties) {
-        return properties.asPairsWithComment("Do not change this generated file. Logging can be configured in the corresponding kubernetes/openshift resource.");
+        return properties.asPairsWithComment("Do not change this generated file. Logging can be configured in the corresponding Kubernetes/OpenShift resource.");
```

Comment:
I would leave just Kubernetes.

Reply:
As in remove the reference to OpenShift all together?

Reply:
I would say that where the code is not related to OpenShift specific stuff, we should just mention Kubernetes.

Reply:
Can I do this in a followup PR Paolo?

Reply:
Addressed :)